.vscode/settings.json

```diff
@@ -1,3 +1,4 @@
 {
-    "editor.insertSpaces": false
-}
\ No newline at end of file
+	"editor.defaultFormatter": "biomejs.biome",
```

Comment:
I would not commit the settings. This could be very individual.
The recommended extension on the other hand are ok because they are not forcing you to anything